### ISSUE TITLE
Add button to change localization/language

### ISSUE THREAD

**Issue by joanise** [ReadAlongs/Studio-Web#156]
We serve English to / and French to /fr but that's not visible anywhere.

EDIT: also Spanish on /es, since we added Spanish since I first created this ticket.

**Comment by deltork** [comment]
It is now available in the settings ( as of version 1.3.x)

**Comment by joanise** [comment]
Not quite, no: this ticket was for Studio-Web. The gear menu lets us change the language of the Web-Component, but not that of the Studio-Web.

**Comment by sergeleger** [comment]
I looked at this briefly, the issue for me was that the development environment does not reflect the production version at https://readalong-studio.mothertongues.org/#/.

In development, English is available as localhost:4200 while the French version is on localhost:4203. MT has a different server configuration (maybe with redirects?) where `/fr` is available.

We could add a conditional language switcher component, that would be available only while navigating MT?

**Comment by joanise** [comment]
Indeed, in production, you have to add `/fr` or `/es` to get the localized studio-web. And what you get is a completely independent app with its own separate memory, so changing language completely resets the state. In that sense, the dev setup is similar, with the three versions not talking to each other.

The minimum feature I imagine is a link that opens the other in a new tab. This could be driven by a config file, in production that config file would say something like "enURL" = "/", "frURL" = "/fr/", etc, and in dev the file would say something like "enURL" = "http://localhost:4200" etc. I'm pretty sure we already have a config file somewhere, so it could be extended.

An nicer option would be in-app toggle without resetting the memory, but we have made design decisions a long time ago that are incompatible with this, so that's not what this Issue is asking for.

**Comment by sergeleger** [comment]
Additionally, 

- add support for toggling to Spanish.
- add support for `/pr-preview` deploys
- add support for `/dev` deploys
